config/development.json

```diff
@@ -1,4 +1,5 @@
 {
   "env": "development",
-  "publicPath": "/"
+  "publicPath": "/v2/",
```

Comment:
Are you sure about that? Wouldn't you have to change how it's mapped in kong if this change comes in?

Reply:
It's only apply to the development environment

Reply:
Ok! 
Thanks
Sorry rs